### PR TITLE
creator tools strings

### PR DESCRIPTION
**Pull request by eoji**
# what
Updating creator tools strings and tests.

# see

### CODE REVIEW

**Comment by eoji** [comment]
[Creator tools copy sweep](https://trello.com/c/2SZIuJp4/421-creator-tools-copy-sweep)

File: app/src/test/java/com/kickstarter/libs/utils/StringUtilsTest.java [hunk 1]
```diff
@@ -31,4 +31,13 @@ public void testIsPresent() {
     assertTrue(StringUtils.isPresent("a"));
     assertTrue(StringUtils.isPresent(" a "));
   }
+
+  @Test
+  public void testSentenceCase() {
```
Comment (eoji) [inline app/src/test/java/com/kickstarter/libs/utils/StringUtilsTest.java]: Let's add a test for the single and empty cases.

File: app/src/main/java/com/kickstarter/ui/viewholders/CreatorDashboardReferrerStatsViewHolder.java [hunk 2]
```diff
@@ -50,10 +54,14 @@ public CreatorDashboardReferrerStatsViewHolder(final @NonNull View view) {
       .compose(observeForUI())
       .subscribe(this::toggleRecyclerViewAndEmptyStateVisibility);
 
-    this.viewModel.outputs.referrerStatsTruncatedTextIsGone()
+    this.viewModel.outputs.referrersTitleIsLimitedCopy()
```
Comment (eoji) [inline app/src/main/java/com/kickstarter/ui/viewholders/CreatorDashboardReferrerStatsViewHolder.java]: Let's rename to `referrersTitleHasTopTen`

File: app/src/main/java/com/kickstarter/ui/viewholders/CreatorDashboardRewardStatsViewHolder.java [hunk 3]
```diff
@@ -56,6 +62,16 @@ public CreatorDashboardRewardStatsViewHolder(final @NonNull View view) {
       .compose(bindToLifecycle())
       .compose(observeForUI())
       .subscribe(gone -> ViewUtils.setGone(this.truncatedTextView, gone));
+
+    this.viewModel.outputs.rewardsTitleIsLimitedCopy()
```
Comment (eoji) [inline app/src/main/java/com/kickstarter/ui/viewholders/CreatorDashboardRewardStatsViewHolder.java]: Rename this one also.
